esmvaltool/recipes/hydrology/recipe_wflow.yml

```diff
@@ -40,7 +40,7 @@ diagnostics:
         mip: day
         preprocessor: rough_cutout
         start_year: 1990
-        end_year: 1990
+        end_year: 2001
```

Comment:
Why change the years?

Reply:
One year of data does not cause any memory error. Also, we run the recipes for at least 10 years in our hydrological applications. As discussed by @nielsdrost and @bouweandela, the recipe should use realistic time length.